### PR TITLE
Info VLR updates

### PR DESCRIPTION
**Pull request by hobu**
span->spacing, extents VLR info, and center point and halfsize

* Rename `span` to `spacing` and adapt language from #26
* Add `extent_vlr_offset` and `extent_vlr_size` from #28
* Add `center_x`, `center_y`, and `halfsize` from #26
* Remove offset positions from `info` VLR
* Add discussion section at bottom of the document
*

### CODE REVIEW

File: DRAFT-SPEC.md [hunk 1]
```diff
@@ -240,6 +258,12 @@ their presence is allowed.
 An Extra Bytes VLR containing that information *MUST* be present
 if extra per-point data is provided.
 
+## Versioning
+
+Should any future incremental versioning of COPC occur, is expected to utilize VLR
```
Comment (abellgithub) [inline DRAFT-SPEC.md:263]: Language is wonky here.

File: DRAFT-SPEC.md [hunk 2]
```diff
@@ -240,6 +258,12 @@ their presence is allowed.
...
+definitions with incremented ``Record ID`` values as the discrimination mechanism. It *MAY*
```
Comment (abellgithub) [inline DRAFT-SPEC.md:264]: I don't understand what this is saying. Perhaps just ditch?

File: DRAFT-SPEC.md (hunk 1 above)
Comment (hobu) [reply DRAFT-SPEC.md]: dropped

File: DRAFT-SPEC.md (hunk 2 above)
Comment (hobu) [reply DRAFT-SPEC.md]: ditched

**Pull request merged**